pyproject.toml

```diff
@@ -2,11 +2,22 @@
 name = "PyCount"
 version = "0.1.0"
 description = "Simple Python project template"
-authors = [""]
+authors = ["Adriana Solis <solisa@allegheny.edu>, Alexis Caldwell <caldwella2@allegheny.edu>, Rachael Harris <harrisr@allegheny.edu>, Ramon Guzman <guzmanr@allegheny.edu>, Ryan Hilty <hiltyr@allegheny.edu>"]
 license = "MIT"
+packages = [
+    { include = "py_counter.py", from="src" },
+    { include = "src/py_counter.py"},
+]
+
+[tool.poetry.scripts]
+PyCount = "src.command_line_interface:cli"
 
 [tool.poetry.dependencies]
 python = ">=3.7,<4.0"
+typer = {extras = ["all"], version = "^0.3.2"}
+rich = "^10.9.0"
+libcst = "^0.4.0"
+black = "^21.12b0"
```

Comment:
`black` should be a dev dependency (added with `poetry add --dev black`)

Reply:
@solisa986